pygments/lexers/tal.py

```diff
@@ -0,0 +1,58 @@
+"""
+    pygments.lexers.uxntal
+    ~~~~~~~~~~~~~~~~~~~~~~
+
+    Lexers for Uxntal
+
+    :copyright: Copyright 2006-2022 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+import re
+import keyword
+
+from pygments.lexer import RegexLexer
+from pygments.token import Comment, Keyword, Name, String, Number, Punctuation, Whitespace, Literal
+
+__all__ = ['TalLexer']
+
+class TalLexer(RegexLexer):
+
+    name = 'Tal'
+    aliases = ['tal', 'uxntal']
+    filenames = ['*.tal']
+    mimetypes = ['text/x-uxntal']
+
+    tokens = {
+        'comment': [
+            (r'(?:(?<=\s)|^)\((?:(?=\s)|$)', Comment.Multiline, '#push'), # nested comments
+            (r'(?:(?<=\s)|^)\)(?:(?=\s)|$)', Comment.Multiline, '#pop'), # nested comments
+            (r'\S+', Comment.Multiline), # comments
+            (r'\s+', Comment.Multiline), # comments
+        ],
+        'root': [
+            (r'\s+', Whitespace), # spaces
+            (r'(?:(?<=\s)|^)\((?:(?=\s)|$)', Comment.Multiline, 'comment'), # comments
+            (r'(?:(?<=\s)|^)(BRK|LIT|INC|POP|DUP|NIP|SWP|OVR|ROT|EQU|NEQ|GTH|LTH|JMP|JCN|JSR|STH|LDZ|STZ|LDR|STR|LDA|STA|DEI|DEO|ADD|SUB|MUL|DIV|AND|ORA|EOR|SFT)2?k?r?(?:(?=\s)|$)', Keyword.Reserved), # instructions
+            (r'(?:(?<=\s)|^)[][{}](?:(?=\s)|$)', Punctuation), # delimiers
+            (r'#([0-9a-f]{2}){1,2}', Number.Hex), # integer
+            (r'"\S+', String), # raw string
```

Comment:
So there is a delimiter but no string content?

In any case, please make sure this is covered in the test file. Also, normally you'd use a lookahead assertion for the whitespace, to keep it highlighted as `Whitespace` without merging it with the `String` token.

Reply:
Good catch! This is a bug, it should be `"\s+` (i.e. a quote followed by one or more non-space characters).

Reply:
Ah, sorry -- the `S` character looked close to an `s` in my browser.